posts/2022-01-17-why-we-combined-graphql-and-a-serverless-distributed-ledger.md

```diff
@@ -0,0 +1,73 @@
+
+Each participant in a Uni has cloud resources created on their behalf. One of the resources is a managed, scalable, resilient, and secure GraphQL API that allows partners to create web, mobile, and backend applications to read data from and write data to a Uni.
+
+While technical readers may be familiar with the concept of a REST API, there may be some among you who are not as familiar with GraphQL. For those of you with GraphQL experience, Vendia Share does take a non-traditional approach in some areas as well.  In this post, we'll dive a bit deeper and explore GraphQL and look at why Vendia selected GraphQL as its primary interface. Through this post, you'll learn more about GraphQL and how it can improve your development experience with Vendia Share.
+
+# What is GraphQL?
+
+[GraphQL was created by Meta / Facebook in 2012](https://engineering.fb.com/2015/09/14/core-data/graphql-a-data-query-language/) as a way to solve problems encountered when rebuilding their native mobile applications. The team was frustrated with the differences between the data they wanted to use in their applications and the server queries that were required and was challenged to pull in data from many disparate sources. They saw the importance of seeing data not just as a resource but as a graph.
+
+GraphQL is a query language for APIs and a runtime for fulfilling those queries with existing data. GraphQL is a [open specification](https://spec.graphql.org/); it is not controlled or managed by a single vendor. GraphQL clients and servers can be written in a [number of programming languages](https://graphql.org/code/). There is no requirement to use particular data stores to persist data.
+
+# What are Some Benefits of GraphQL?
+
+There are several notable benefits of using GraphQL.
+
+## Data Precision
+
+GraphQL allows clients to request exactly what they need and nothing more. This means that developers don't have to worry about over-fetching and under-fetching data. It's not uncommon to experience these issues when using other API strategies like REST. Eliminating these minimizes the number of network requests and client memory use required to satisfy an application's needs.
+
+## Many Resources in One Request
+
+GraphQL allows applications to get all the data they need - even if they are different types - in a single request. This means fewer requests and, ultimately, more responsive applications for end users.
+
+## Strong Typing
+
+GraphQL is strongly typed. This means that developers can know what is required by a GraphQL server to satisfy a request without having to be planning for type mismatches.
+
+## Subscriptions
+
+Subscriptions provide a mechanism for fetching data from a GraphQL server over a long-lived connection (usually via WebSocket). They are a handy mechanism for providing updates to GraphQL clients when new objects are created or existing objects have updates made.
```

Comment:
You use `resources` in the `Many Resources in One Request` but `object` in this section.  I'd pick a single term for consistency - entity, resource, object (I'd favor `resource` since I believe that's most inline with GraphQL docs, and easy for those with more REST familiarity to grasp).